### DISCUSSION TITLE
Nested Workflow Names on Left Pane

### DISCUSSION THREAD

**Discussion by s0undt3ch** [community/community#44918] [Actions]
### Select Topic Area

Bug

### Body

While nesting workflows, the `/` concatenated names seen on the right pane, do not show up on the left pane, which makes it all too confusing.

![Screenshot_20230120_185908](https://user-images.githubusercontent.com/300048/213787597-c75474ed-ddb5-4607-957a-b629834d4c4b.png)

I'm guessing this is a bug?
Any work around?

**Comment by s0undt3ch** [comment]
According to the [docs](https://docs.github.com/en/actions/using-workflows/reusing-workflows#limitations), we're limited to 4 levels of nesting, ie, the top level and 3 more.

GitHub should render what it supports.

**Comment by cfoch** [comment]
Any workaround here?
